package.json

```diff
@@ -5,10 +5,11 @@
   "type": "module",
   "main": "./dist/compound-web.js",
   "module": "./dist/compound-web.js",
+  "sideEffects": false,
   "exports": {
     ".": {
-      "require": "./dist/compound-web.cjs",
-      "import": "./dist/compound-web.js",
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.js",
```

Comment:
Because we changed the `fileName` pattern, the output files match the input files structure in `src/`